database/Image.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+mongoose.Promise = global.Promise;
+
+const imageSchema = new mongoose.Schema({
+  restaurantId: Number,
+  photoId: Number,
+  photoTitle: String,
+  photoUrl: String
+});
+
+const Image = mongoose.model('Image', imageSchema);
+
+var findAll = () => {
+  return new Promise((resolve, reject) => {
+    Image.find()
+      .exec((err, data) => {
+        resolve(data);
+      });
+  });
```

Comment:
`.exec()` already gives us a promise, so no need to return a new promise here. Reference: https://mongoosejs.com/docs/promises.html#queries-are-not-promises

Reply:
Cool! Thank you Miao!